spec/features/issues_spec.rb

```diff
@@ -274,7 +274,7 @@
 
           it 'presents the list of hosts affected by a given issue'  do
             expect(find('.secondary-sidebar-content')).to have_content('10.0.0.1')
-            expect(find('.secondary-sidebar-content')).to have_content('10.0.0.2', count: 5)
+            expect(find('.secondary-sidebar-content')).to have_content('10.0.0.2', count: 3)
```

Comment:
But why did the count change if nothing else changed? 🧐

Reply:
This actually brings it back to the way it is in `main`. This was a temporary change in the base branch to pass specs before the changes from this PR are merged in.